examples/graph/plot_DAG_layouts.py

```diff
@@ -0,0 +1,74 @@
+"""
+======================
+DAG in Various Layouts
+======================
+
+Creates a Directed Acyclic Graph and shows the Graph in various layouts. 
+This can give an idea as to which layout can be chosen for these types of datasets.
+
+"""
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+
+def DAG():
```

Comment:
Just a nit, but IMO this example would be more clear *without* encapsulating the code into functions. Wrapping things into functions can be very useful, especially when you intend for code to be reused, but in this case it adds an extra layer and makes the example less clear because you have to read the function definitions, then find where they're called, and re-map the execution order in your head! Of course, in this case it's very simple to do so, but only serves to show that wrapping things up into functions is unnecessary.

Reply:
Yes I understand. I tend to always right code in terms of functions to make it easier for me 😅. I will make the necessary edits in the code. Thanks for the suggestion!